core/domain/translation_domain.py

```diff
@@ -13,7 +13,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Models for machine translations retrieved from Google Cloud Translate."""
+
+"""Domain objects for machine translations retrieved from Google Cloud
```

Comment:
`Domain objects related to translations.`

(We might put other stuff in this file in future.)

Reply:
Fixed.